src/components/publishing/display/canvas/text.tsx

```diff
@@ -0,0 +1,112 @@
+import { compact, map } from "lodash"
+import React from "react"
+import styled, { StyledFunction } from "styled-components"
+import { pMedia } from "../../../helpers"
+import Fonts from "../../fonts"
+
+interface DisplayCanvasTextProps extends React.HTMLProps<HTMLDivElement> {
+  disclaimer?: any
+  unit: any
+}
+
+interface DivProps extends React.HTMLProps<HTMLDivElement> {
+  layout?: any
+  isSlideshowWithCaption?: boolean
+}
+
+const DisplayCanvasText: React.SFC<DisplayCanvasTextProps> = props => {
+  const { disclaimer, unit } = props
+  const isSlideshowWithCaption = unit.layout === "slideshow" && compact(map(unit.assets, "caption")).length
```

Comment:
Would the text layout have a slideshow?

Reply:
Hm actually I seem to be misunderstanding how this component gets used.

Reply:
This is the text of the unit, not a text layout.  It is used in all layouts.